content/docs/apps/static-egress.md

```diff
@@ -9,17 +9,14 @@ If your cloud.gov-hosted application needs access to an external service, that s
 
 ## Securing communications with external services
 
-You can set up your cloud.gov application to communicate with any external service available over the internet. To ensure only your application can communicate with the external service, the service should provide application-level authentication.
+You can set up your cloud.gov application to communicate with any external service available over the internet. To ensure only your application can communicate with the external service, the service should be configured to authenticate each client using client-specific credentials.
 
-If the external service also requires IP filtering, you can allow access from the [cloud.gov egress addresses](#cloud-gov-egress-addresses).
+## Restricting network exposure of an external service 
 
-**Note:** Filtering by IP address will only ensure that traffic is originating from a cloud.gov application, but it provides no guarantee that it is from your application. Applications from other agencies and [sandbox accounts]({{< relref "overview/pricing/free-limited-sandbox.md" >}}) will also originate from the same addresses.
-
-
-## cloud.gov egress addresses
```

Comment:
I want some anchor right above the IP address so we can link directly to them, and not this page in general when folks do ask for them.